readme.md

```diff
@@ -25,8 +25,8 @@ We love to receive contributions. You can submit bug reports [here](https://gith
 ```
 Copyright (C) 2015-2016 Winfried Simon
 
-This software may be used under the terms of the GNU Lesser General 
-Public License version 2.1 as published by the Free Software Foundation 
+This software may be used under the terms of the GNU Lesser General
+Public License version 2.1 as published by the Free Software Foundation
```

Comment:
What are the changes here?

Reply:
I changed the Markdown document's ``pyqt5`` to ``pyqt6`` to reflect the latest updates. My editor automatically removed the trailling white spaces at the line end, causing these two lines to be changed.